plugins/parodos/src/components/ParodosPage/TabLabel.tsx

```diff
@@ -17,19 +17,21 @@ export interface TabLabelProps {
   children: ReactNode;
 }
 
-export function TabLabel({
-  children,
+export const tabLabelCreator = ({
   icon,
   highlighted,
-  ...props
-}: TabLabelProps) {
-  const styles = useStyles();
+}: {
+  icon: ReactNode;
+  highlighted: boolean;
+}) =>
+  forwardRef<HTMLSpanElement, TabLabelProps>(({ children, ...props }, ref) => {
+    const styles = useStyles();
 
-  return (
-    <span {...props}>
-      {icon}
-      {highlighted && <StarIcon className={styles.highlightedTab} />}
-      {children}
-    </span>
-  );
-}
+    return (
+      <span {...props} ref={ref}>
```

Comment:
Where do we use this `ref`?

Reply:
> Where do we use this `ref`?

We don't but material-ui does, it leaves a huge warning error message in the console

![ref](https://user-images.githubusercontent.com/118328/227200760-8eccaa3b-1253-4b0e-b84b-470aa498f62c.png)